libs/ui/src/scss/variable.scss

```diff
@@ -0,0 +1,11 @@
+$k-accent: #ff7ac3;
```

Comment:
Is this from figma file? cc @exezbcz

Reply:
yes sir
![image](https://user-images.githubusercontent.com/90852205/197561132-1fffde67-c009-43d0-a0e3-b3dea59209ab.png)